File: CareConnect-client/careConnect-client/src/app/models/physicianTimeOffResult.ts
export interface PhysicianTimeOffResult {
  physicianTimeOffId: number;
  physicianId: number;
  startDateTime: Date;
  endDateTime: Date;
  isAllDay: boolean;
  reason: string | null;
  notes: string | null;
  createdAt: Date | null;
  updatedAt: Date | null;
}

File: CareConnect/Models/Database/results/PhysicianTimeOffResult.cs
namespace CareConnect.Models.Database.results
{
    public class PhysicianTimeOffResult
    {
        public int PhysicianTimeOffId { get; set; }
        public int PhysicianId { get; set; }
        public DateTime StartDateTime { get; set; }
        public DateTime EndDateTime { get; set; }
        public bool IsAllDay { get; set; }
        public string? Reason { get; set; }
        public string? Notes { get; set; }
        public DateTime? CreatedAt { get; set; }
        public DateTime? UpdatedAt { get; set; }
    }
}